$MirOS: ports/www/php/patches/patch-ext_calendar_calendar_c,v 1.1.7.1 2005/03/18 16:00:00 tg Exp $
$OpenBSD: patch-ext_calendar_calendar_c,v 1.1.1.1 2004/10/02 11:32:37 robert Exp $
--- ext/calendar/calendar.c.orig	Thu Apr 14 15:36:23 2005
+++ ext/calendar/calendar.c	Wed Jan 25 17:54:43 2006
@@ -304,7 +304,7 @@ PHP_FUNCTION(cal_from_jd)
 
 	calendar->from_jd(jd, &year, &month, &day);
 
-	sprintf(date, "%i/%i/%i", month, day, year);
+	snprintf(date, sizeof date, "%i/%i/%i", month, day, year);
 	add_assoc_string(return_value, "date", date, 1);
 
 	add_assoc_long(return_value, "month", month);
